src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const DAY_COST = 40;
+
+  if (days < SHORT_TERM) {
+    return days * DAY_COST;
+  };
+
+  if (days < LONG_TERM) {
+    return days * DAY_COST - SHORT_TERM_DISCOUNT;
+  };
+
+  return days * DAY_COST - LONG_TERM_DISCOUNT;
```

Comment:
Let's create a variable for `days * DAY_COST` in order not to duplicate it three times